src/main/resources/templates/navbar.html

```diff
@@ -1,25 +1,36 @@
 <html xmlns:th="http://www.thymeleaf.org"
-      xmlns:sec="https://www.thymeleaf.org/thymeleaf-extras-springsecurity6">
-<nav class="site-header sticky-top py-1">
-    <div class="col-sm-12 col-md-6 col-lg- 1"><a href="/"><h1><span style="color: #ff008c">You</span><span style="color: #6ed2f0">rGr</span><span style="color: #3cfa50">ade</span></h1></a></div>
+      xmlns:sec="https://www.thymeleaf.org/thymeleaf-extras-springsecurity6" xmlns="">
+<link rel="stylesheet" href="../css/styles.css">
+<body data-spy="scroll" data-target="#spy-scroll-id">
+<nav id="spy-scroll-id" class="site-header sticky-top py-1">
+    <div class="col-sm-12 col-md-6 col-lg- 1"><a href="/"><h1><span style="color: #ff008c">You</span><span
+            style="color: #6ed2f0">rGr</span><span style="color: #3cfa50">ade</span></h1></a></div>
     <div class="container d-flex flex-column flex-md-row justify-content-between">
-        <div class="col-sm-12 col-md-6 col-lg-2"><a href="/">home</a></div>
+        <div class="col-sm-12 col-md-6 col-lg-2"><a  href="/">home</a></div>
```

Comment:
is there a reason for extra spaces in front of "href"? i don't think they make sense

Reply:
yes it's typo, fixed